src/EditorFeatures/Core/Implementation/IntelliSense/AsyncCompletion/CompletionSource.cs

```diff
@@ -49,8 +49,8 @@ internal class CompletionSource : ForegroundThreadAffinitizedObject, IAsyncCompl
         private static readonly EditorOptionKey<bool> NonBlockingCompletionEditorOption = new EditorOptionKey<bool>(NonBlockingCompletion);
 
         // Use CWT to cache data needed to create VSCompletionItem, so the table would be cleared when Roslyn completion item cache is cleared.
-        private static readonly ConditionalWeakTable<RoslynCompletionItem, VSCompletionItemData> s_roslynItemToVsItemData =
-            new ConditionalWeakTable<RoslynCompletionItem, VSCompletionItemData>();
+        private static readonly ConditionalWeakTable<RoslynCompletionItem, StrongBox<VSCompletionItemData>> s_roslynItemToVsItemData =
+            new ConditionalWeakTable<RoslynCompletionItem, StrongBox<VSCompletionItemData>>();
```

Comment:
Hmm, I had thought that VSCompletionItemData was a struct for some reason. Why did we need the StrongBox again if it's a class?

Reply:
(I know I mentioned it in my previous review, but I'm not remembering why I thought that was necessary. It's also Friday and I'm a bit sleepy so it could be something dumb.)

Reply:
Oh, right, I forgot to make it a struct in this PR (but done it in a stale branch which I copied code from)

Reply:
@genlu: ah, that's what it was. thanks.